vortex-api/src/main/java/mil/army/usace/hec/vortex/io/NetcdfDataReader.java

```diff
@@ -395,6 +397,8 @@ private List<ZonedDateTime[]> getTimeBounds(GridCoordSystem gcs) {
                     zonedDateTimes[1] = zonedDateTimes[0].plusHours(1);
                 } else if (fileName.matches("ge.*\\.pgrb2.*\\.0p.*\\.f.*\\..*")) {
                     zonedDateTimes[1] = zonedDateTimes[0].plusHours(3);
+                } else if (fileName.matches("prec.[0-9]{4}.nc")) {
+                    zonedDateTimes[1] = convert(tAxis.getCalendarDate(time)).plusDays(1);
```

Comment:
I would use `zonedDateTimes[0].plusDays(1);` here since the start date has already been retrieved.

Reply:
Got it.  Just updated in the latest commit.